tests/boardfarm_plugins/boardfarm_prplmesh/devices/prplmesh_base.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+
+import pexpect
+
+from boardfarm.devices import debian
+
+
+class PrplMeshBase(debian.DebianBox):
```

Comment:
> I tried with LinuxDevice and so far at least, it works.

I've checked - no difference when using `LinuxDevice` instead of `DebianBox`.
Replaced `DebianBox` by `LinuxDevice`.